src/lib/components/market/item_row_wrapper.ts

```diff
@@ -158,7 +160,8 @@ export class ItemRowWrapper extends FloatElement {
 
             return html`
                 <div class="float-row-wrapper">
-                    Float: ${this.itemInfo.floatvalue.toFixed(14)} ${renderClickableRank(this.itemInfo)}<br />
+                    ${this.renderFloatBar()} Float: ${this.itemInfo.floatvalue.toFixed(14)}
```

Comment:
nit: Would be cleaner to wrap the `Float: ... ${renderClickableRank(...)}` portion in a `span`